### ISSUE TITLE
pls help, why can't show Image, when it in table cell. 

### ISSUE THREAD

**Issue by hetz** [flexumio/beautiful-docx#29]
I have html code
```
export const exampleText = `
<table>
    <tr>
        <td><img src="https://img-s.msn.cn/tenant/amp/entityid/AA1b4Jpr.img" /></td>
    </tr>
</table>
<img src="https://img-s.msn.cn/tenant/amp/entityid/AA1b4Jpr.img"
    alt="image.png" style="text-align: center;" />
`;
```

I get this docx

I try fix it, but code is very ugly.
![image](https://github.com/flexumio/beautiful-docx/assets/820141/cdef819f-51d3-4dcd-890c-5f9c413a153d)

**Comment by hetz** [comment]
I find docx example : https://github.com/dolanmiu/docx/blob/master/demo/24-images-to-table-cell.ts
When image in table cell, must warp p. 

When Image transformToDocx ,I can't get parent element ,so  I can't dynamic return ...

**Comment by kotsanolena** [comment]
Hi, @hetz 

We have updated and fixed the issue you reported in our latest package release.

After updating, the issue you encountered should be resolved. If you have any further questions or encounter any other issues, feel free to let us know.

**Comment by hetz** [comment]
I don't find commit about this fix,  pls show me .

**Comment by kotsanolena** [comment]
Hi @hetz, we have fixed the issue in the following commits: [6dbe08c](https://github.com/flexumio/beautiful-docx/commit/6dbe08ce0cd19406376a02c2b57b002e2674a33b), [0e347ac](https://github.com/flexumio/beautiful-docx/commit/0e347ac3233f5f486928de2dbbdf3863e503f785) and [9607b53](https://github.com/flexumio/beautiful-docx/commit/9607b530d45662ff47a02f633ced5531d9e24c42)
